Resources/Prototypes/_ES/Fills/Crates/caches.yml

```
@@ -6,6 +7,46 @@
   suffix: ES
   abstract: true
 
+# Assassin
+- type: entity
+  parent: ESCrateCache
+  id: ESCrateCacheTraitorAssassin
+  suffix: ES, Assassin Cache
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:NestedSelector
+        tableId: ESTraitorCacheAssassin
+
+- type: entityTable
+  id: ESTraitorCacheAssassin
+  table: !type:AllSelector
+    children:
+    - id: ESWeaponPistol9mmSilenced
+    - id: ESMagazine9mm
+      amount: 2
+    - id: ESSleepyPen
+      amount: 2
```

```suggestion
  table:
    all:
    - id: ESWeaponPistol9mmSilenced
    - id: ESMagazine9mm
      amount: 2
    - id: ESSleepyPen
      amount: 2
```
minor table cleanup like this can be done pretty generally across this PR